### PR TITLE
Update Templates

### PR DESCRIPTION
**Pull request by rohit-dimagi**

### CODE REVIEW

**Review by snopoke — APPROVED** [review]

File: roles/filebeat/templates/filebeat.yml.j2 [hunk 1]
```diff
@@ -1,6 +1,12 @@
 filebeat.inputs:
-{{ filebeat_inputs | to_nice_yaml }}
-
+{% for item in filebeat_inputs %}
+- type: log
+  paths:
+    - {{ item['paths'] }}
```
Comment (snopoke) [inline roles/filebeat/templates/filebeat.yml.j2]: mm, won't this need to be a for loop? or else `paths: {{ item['paths'] | to_yaml }}`

Comment (rohit-dimagi) [reply roles/filebeat/templates/filebeat.yml.j2]: In case we have multiple paths ?

Comment (snopoke) [reply roles/filebeat/templates/filebeat.yml.j2]: yea, or else rename this to `path` so it's not plural

Comment (rohit-dimagi) [reply roles/filebeat/templates/filebeat.yml.j2]: Renamed it to path.

**Pull request merged**